src/redux/reducers/index.js

```diff
@@ -3,13 +3,16 @@ import AuthReducers from './authReducers';
 import sources from './sourcesReducer';
 import organizations from './dictionaryReducer';
 import concepts from './ConceptReducers';
+import dictionaries from './dictionaryReducers';
+
```

Comment:
Aren't we leaving many lines than required here?

Reply:
I will deal with that thank you